Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,28 @@ public extension FileManager {
         return nil
     }
 
+    /// Creates a unique directory for saving temporary files.
+    ///
+    /// The directory can be used to create multiple temporary files used for a common purpose.
+    ///
+    ///     let tempDirectory = try fileManager.temporaryFileDirectory()
+    ///     let tempFile1URL = tempDirectory.appendingPathComponent(ProcessInfo().globallyUniqueString)
+    ///     let tempFile2URL = tempDirectory.appendingPathComponent(ProcessInfo().globallyUniqueString)
+    ///
+    /// - Returns: A URL to a new directory for saving temporary files.
+    /// - Throws: An error if a temporary directory cannot be found or created.
+    public func temporaryFileDirectory() throws -> URL {
```

Comment:
My only concern here is that the naming doens't express the idea that a new directory is going to be created every time you call this. Even with just being a method kinda give this idea, I'm not sure if it's clear enough. What do you think? Maybe `makeTemporaryFileDirectory`?

Reply:
I was thinking a similar thing. How about `createTemporaryDirectory`? That would make it in line with the `create` parameter of `url(for:in:appropriateFor:create:)`.